openrouteservice/src/main/java/org/heigit/ors/api/controllers/IsochronesAPI.java

```diff
@@ -110,14 +112,16 @@ public ResponseEntity<Object> handleMissingParams(final MissingServletRequestPar
         return errorHandler.handleStatusCodeException(new MissingParameterException(IsochronesErrorCodes.MISSING_PARAMETER, e.getParameterName()));
     }
 
-    @ExceptionHandler({HttpMessageNotReadableException.class, HttpMessageConversionException.class})
+    @ExceptionHandler({HttpMessageNotReadableException.class, ConversionFailedException.class, HttpMessageConversionException.class, Exception.class})
     public ResponseEntity<Object> handleReadingBodyException(final Exception e) {
         final Throwable cause = e.getCause();
         if (cause instanceof UnrecognizedPropertyException) {
             return errorHandler.handleUnknownParameterException(new UnknownParameterException(IsochronesErrorCodes.UNKNOWN_PARAMETER, ((UnrecognizedPropertyException) cause).getPropertyName()));
         } else if (cause instanceof InvalidFormatException) {
             return errorHandler.handleStatusCodeException(new ParameterValueException(IsochronesErrorCodes.INVALID_PARAMETER_FORMAT, ((InvalidFormatException) cause).getValue().toString()));
-        } else if (cause instanceof InvalidDefinitionException) {
+        } else if (cause instanceof ConversionFailedException) {
+            return errorHandler.handleStatusCodeException(new ParameterValueException(IsochronesErrorCodes.INVALID_PARAMETER_FORMAT, Objects.requireNonNull(((ConversionFailedException) cause).getValue()).toString()));
```

Comment:
Objects.requireNonNull is not necessary here as instanceof do already the no null check
```suggestion
return errorHandler.handleStatusCodeException(new ParameterValueException(IsochronesErrorCodes.INVALID_PARAMETER_FORMAT, ((ConversionFailedException) cause).getValue().toString()));
```